src/ABIForm.tsx

```diff
@@ -4,10 +4,11 @@ import React, { useMemo, useState } from 'react';
 
 import { ABI, abiSchema } from './types/index';
 import {
+  extractEnumsFromABI,
   extractFunctionFromRawAbi,
   extractStructFromABI,
-  segregateViewAndExternalFunctions,
-} from './types/helper';
+  segregateViewAndExternalFunctions
+} from "./types/helper";
```

Comment:
@prxgr4mm3r I guess your prettier extension is not setup properly. I see it's changing single quotes to double quotes and removing trailing comma as well. Could you please fix it!